src/ResourceManager/AnalysisServices/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Current Release
 
+## Version 0.5.1
```

Comment:
@taiwu we will take care of the versioning based on the public API changes made to a cmdlet. You can remove the `## Version 0.5.1` header and put the below snippet under the `## Current Release` header